apps/hash-realtime/src/config.ts

```diff
@@ -1,38 +1,42 @@
 import { getRequiredEnv } from "@local/hash-backend-utils/environment";
 import { Logger } from "@local/hash-backend-utils/logger";
-import { QueueProducer } from "@local/hash-backend-utils/queue/adapter";
-import { RedisQueueProducer } from "@local/hash-backend-utils/queue/redis";
+import {
+  Entity,
+  EntityType,
+  PropertyType,
+} from "@local/hash-backend-utils/pg-tables";
 import { supportedRealtimeTables } from "@local/hash-backend-utils/realtime";
-import { AsyncRedisClient } from "@local/hash-backend-utils/redis";
+import { RedisConfig, setupRedisClient } from "@local/hash-backend-utils/redis";
+import { RedisStreamProducer } from "@local/hash-backend-utils/stream/redis";
 
 // The tables to monitor for changes
 export const MONITOR_TABLES = supportedRealtimeTables.map(
   (table) => `public.${table}`,
 );
 
+const redisConfig: RedisConfig = {
+  host: process.env.HASH_REDIS_HOST ?? "localhost",
+  port: parseInt(process.env.HASH_REDIS_PORT ?? "6379", 10),
```

Comment:
Yeah, it is needless. It does allow our defaults to be easier to discover, though. Perhaps we should have a `getRequiredEnv` equivalent called `getOptionalEnv` or something that can be defaulted and do the inner ternary there?

Reply:
`getEnvWithDefault` or something 😅

Reply:
After thinking about this, if we introduce `getEnvWithDefault` we'd still have a problem of parsing the string value as an int again since that wouldn't be done in `getEnvWithDefault` itself. Wouldn't change anything about the original comment :(

Reply:
You could pass a `convertType` function into `getEnvWithDefault` 😂...

definitely just theorycrafting amusing things here and we shouldn't change anything